### PR TITLE
902663: Updated UG for TabView control

### PR DESCRIPTION
**Pull request by SwethavasanthSF3977**

### CODE REVIEW

File: MAUI/TabView/Selection-Indicator-Customization.md [hunk 1]
```diff
@@ -163,4 +163,22 @@ The [IndicatorWidthMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Tab
 
 {% endtabs %}
 
-![Selection Indicator Background.](images/IndicatorWidthMode_Stretch.png) 
\ No newline at end of file
+![Selection Indicator Background.](images/IndicatorWidthMode_Stretch.png) 
+
+## Indicator CornerRadius
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Selection-Indicator-Customization.md:168]: Indicator's corner radius

File: MAUI/TabView/Selection-Indicator-Customization.md [hunk 2]
```diff
@@ -163,4 +163,22 @@ The [IndicatorWidthMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Tab
...
+    <tabView:SfTabView IndicatorCornerRadius ="5">
+{% endhighlight %}
+
+{% highlight C# %}
+     tabView.IndicatorCornerRadius  = 5;
+{% endhighlight %}
+
+{% endtabs %} 
+
+![Selection Indicator Background.](images/IndicatorCornerRadius.png)
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Selection-Indicator-Customization.md:184]: Selection Indicator Corner Radius.

File: MAUI/TabView/Tab-Bar-Customization.md [hunk 3]
```diff
@@ -69,6 +69,53 @@ N> It is recommended to set the TabBarHeight as 72 while displaying the image an
 
 {% endtabs %}
 
+## Customize the Tab Header Alignment
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Tab-Bar-Customization.md:72]: Customize the tab header text alignment

File: MAUI/TabView/Tab-Item-Customization.md [hunk 4]
```diff
@@ -478,4 +478,24 @@ tabView.IsScrollButtonEnabled = true;
 
 {% endtabs %}
 
-![TabView Scroll Mode](images/TabViewScroll.gif) 
\ No newline at end of file
+![TabView Scroll Mode](images/TabViewScroll.gif) 
+
+## ContentTransitionDuration
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Tab-Item-Customization.md:483]: Content transition duration

File: MAUI/TabView/Tab-Item-Customization.md [hunk 5]
```diff
@@ -478,4 +478,24 @@ tabView.IsScrollButtonEnabled = true;
...
+
+When the selected index of the control is changed, you can set the speed of the animation of SelectionIndicator by setting the `ContentTransitionDuration` property in Tab View.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Tab-Item-Customization.md:485]: Modify the content, as we discussed

File: MAUI/TabView/Selection-Indicator-Customization.md (hunk 1 above)
Comment (SwethavasanthSF3977) [reply MAUI/TabView/Selection-Indicator-Customization.md]: changed title

File: MAUI/TabView/Selection-Indicator-Customization.md (hunk 2 above)
Comment (SwethavasanthSF3977) [reply MAUI/TabView/Selection-Indicator-Customization.md]: Changed title

File: MAUI/TabView/Tab-Bar-Customization.md (hunk 3 above)
Comment (SwethavasanthSF3977) [reply MAUI/TabView/Tab-Bar-Customization.md]: changed title

File: MAUI/TabView/Tab-Item-Customization.md (hunk 4 above)
Comment (SwethavasanthSF3977) [reply MAUI/TabView/Tab-Item-Customization.md]: Changed title

File: MAUI/TabView/Tab-Item-Customization.md (hunk 5 above)
Comment (SwethavasanthSF3977) [reply MAUI/TabView/Tab-Item-Customization.md]: Content modified

File: MAUI/TabView/Tab-Item-Customization.md [hunk 6]
```diff
@@ -478,4 +478,24 @@ tabView.IsScrollButtonEnabled = true;
 
 {% endtabs %}
 
-![TabView Scroll Mode](images/TabViewScroll.gif) 
\ No newline at end of file
+![TabView Scroll Mode](images/TabViewScroll.gif) 
+
+## Content transition duration
+
+We can customize the duration of the animation when changing the selected index of the SelectionIndicator in the Tab View by setting the `ContentTransitionDuration` property.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Tab-Item-Customization.md:485]: You can customize the animation duration when changing the selected index of the Tab View by setting the `ContentTransitionDuration` property.

File: MAUI/TabView/Selection-Indicator-Customization.md [hunk 7]
```diff
@@ -163,4 +163,22 @@ The [IndicatorWidthMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Tab
 
 {% endtabs %}
 
-![Selection Indicator Background.](images/IndicatorWidthMode_Stretch.png) 
\ No newline at end of file
+![Selection Indicator Background.](images/IndicatorWidthMode_Stretch.png) 
+
+## Indicator's corner radius
+
+The selection Indicator can be customized using `CornerRadius` property, which is type of CornerRadius.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Selection-Indicator-Customization.md:170]: You can customize the corner radius of the selection indicator using the CornerRadius property in the Tab View.

File: MAUI/TabView/Tab-Item-Customization.md (hunk 6 above)
Comment (SwethavasanthSF3977) [reply MAUI/TabView/Tab-Item-Customization.md]: Added Changes

File: MAUI/TabView/Selection-Indicator-Customization.md (hunk 7 above)
Comment (SwethavasanthSF3977) [reply MAUI/TabView/Selection-Indicator-Customization.md]: Added changes

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]

**Pull request merged**